mesonbuild/scripts/coverage.py

```diff
@@ -85,11 +88,11 @@ def coverage(outputs, source_root, subproject_root, build_root, log_dir):
                                    os.path.join(source_root, '*'),
                                    '--rc', 'lcov_branch_coverage=1',
                                    '--output-file', covinfo])
-            # Remove all directories inside subproject dir
+            # Remove all files/directories defined as excludes
             subprocess.check_call([lcov_exe,
-                                   '--remove', covinfo,
-                                   os.path.join(subproject_root, '*'),
-                                   '--rc', 'lcov_branch_coverage=1',
+                                   '--remove', covinfo] +
+                                   lcov_excludes +
```

Comment:
__[Flake8]__ [E127] continuation line over-indented for visual indent

<sub>posted by Sider</sub>